templates/corporate/upgrade.html

```diff
@@ -24,6 +24,16 @@ <h1>🚀 {% trans %}Upgrade to {{ plan }}{% endtrans %}</h1>
                 </div>
                 {% endif %}
 
+                {% if is_demo_organization %}
+                <div class="demo-organization-warning">
+                    Demo organizations are not eligible for upgrades. Please
+                    <a href="/help/demo-organizations#converting-a-demo-organization-to-a-regular-organization">
+                        convert your demo organization
+                    </a>
+                    before upgrading.
+                </div>
```

Comment:
Hm, "eligible for upgrades" sounds a bit like we offer some kind of upgrade service, as opposed to the user upgrading their org. How about something like this?

>  Demo organizations cannot be upgraded to Zulip Cloud Standard. Please start by <a href="/help/demo-organizations#converting-a-demo-organization-to-a-regular-organization">
                        converting your demo organization</a>.

Reply:
<details>
<summary>screenshot of your updated text</summary>

![Screenshot from 2022-08-23 15-06-19](https://user-images.githubusercontent.com/63245456/186165651-5816cb4e-192d-4ad8-8265-466d2256ed86.png)
</details>

<details>
<summary>screenshot of your updated text with small addition</summary>

![Screenshot from 2022-08-23 15-01-21](https://user-images.githubusercontent.com/63245456/186164689-b2af9c04-48e0-43ec-a9e6-414c32d4cd0c.png)
</details>

If we want it all to be one line, how about:

> Cannot upgrade to Zulip Cloud Standard until you[ convert your demo organization ](https://zulip.com/help/demo-organizations) to a permanent organization.

<details>
<summary>screenshot of one-line text</summary>

![Screenshot from 2022-08-23 15-01-46](https://user-images.githubusercontent.com/63245456/186164794-62f34d5c-0511-46b9-a7ef-bd5ec3e07fb9.png)
</details>

Reply:
Hmm, I think my favorite is the long two-line text (with your addition), since it's the most clear. I think it's fine to have the banner be a bit long.

Reply:
Updated for that version.